python/tvm/topi/hexagon/utils.py

```diff
@@ -150,4 +156,91 @@ def get_layout_transform_fn(layout):
         return nc_2048_2d
     if layout == "nhwc-8h8w32c-2d":
         return nhwc_8h8w32c_2d
+    if layout == "n11c-2048c-2d":
+        return n11c_2048c_2d
     raise RuntimeError(f"Unexpected layout '{layout}'")
+
+
+def get_fixed_point_value(flp, dtype="int16"):
```

Comment:
Could we have a few unit tests for this function?  It's sufficiently complicated that the code isn't obviously correct just from reading it.

Reply:
I'll look into adding some test cases. Thanks!